common/changes/sparo/feat-support_add_profile_in_checkout_2024-02-23-08-25.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "sparo",
+      "comment": "feat: support --add-profile in sparo checkout",
```

```suggestion
      "comment": "Improve \"sparo checkout\" to support the --add-profile parameter",
```
Use friendly language instead of "nerd voice" for CHANGELOG.md text

https://rushjs.io/pages/best_practices/change_logs/